### ISSUE TITLE
Please add all of the other settings "obsidian paste image rename" has

### ISSUE THREAD

**Issue by sidney-eliot** [musug/obsidian-paste-png-to-jpeg#17]
I love the png to jpeg conversion, something that "obsidian paste image rename" lacks.

Please add all the other settings "obsidian paste image rename" has, otherwise this extension is quite unusable, for me at least.

To achieve that with the least effort I would assume forking the other projects branch is the best solution and then adding your png to jpeg conversion code.

**Comment by juanbretti** [comment]
Are you referring to this plugin?
https://github.com/reorx/obsidian-paste-image-rename
https://forum.obsidian.md/t/paste-image-rename-plugin/35480

**Comment by sidney-eliot** [comment]
Yes the one from reorx, I'm deploying my obsidian to the web, so I need small image files, making jpeg essential for me.
